src/messages/MessageElement.cpp

```diff
@@ -188,15 +188,15 @@ void TextElement::addToContainer(MessageLayoutContainer &container,
         for (Word &word : this->words_)
         {
             auto getTextLayoutElement = [&](QString text, int width,
-                                            bool trailingSpace) {
```

Comment:
The element which gets creates *is* not a trailing space it *has* a trailing space. `hasTrailingSpace` would be more fitting.